tests/e2e/instance_test.go

```diff
@@ -56,14 +55,15 @@ func TestConsumerDemocracyTestSuite(t *testing.T) {
 	suite.Run(t, democSuite)
 }
 
-// Executes the standard group of ccv tests against a generic consumer app.go implementation and Gaia as the provider.
-func TestCCVTestSuiteGaia(t *testing.T) {
-
-	// Pass in concrete app types that implement the interfaces defined in /testutil/e2e/interfaces.go
-	ccvSuite := e2e.NewCCVTestSuite[*gaiaApp.GaiaApp, *appConsumer.App](
-		// Pass in ibctesting.AppIniters for gaia (provider) and consumer.
-		icstestingutils.GaiaAppIniter, icstestingutils.ConsumerAppIniter, []string{})
-
-	// Run tests
-	suite.Run(t, ccvSuite)
-}
+// TODO: Add back when Gaia updated
+//// Executes the standard group of ccv tests against a generic consumer app.go implementation and Gaia as the provider.
+//func TestCCVTestSuiteGaia(t *testing.T) {
```

Comment:
Do these tests still fail with https://github.com/cosmos/gaia/releases/tag/v8.0.0-rc1?

Reply:
RC-1 doesn't have ICS integrated. I can add these back in targeting the most recent commit of https://github.com/cosmos/gaia/pull/1938 ...but ICS will actually need to target Lambda.